cpa-repo/src/main/kotlin/no/nav/emottak/cpa/Routes.kt

```diff
@@ -224,18 +230,21 @@ fun Route.validateCpa(cpaRepository: CPARepository) = post("/cpa/validate/{$CONT
             )
         )
     } catch (ebmsEx: EbmsException) {
+        // TODO: Event-logging feil?
```

Comment:
Disse TODOene er ting som burde bli avgjort i stedet for å skrives ned. Oppgaven er vel ikke ferdig før dette blir avgjort uansett, ved mindre vi ønsker å comitte uferdig kode inn til main - som har vært en uting vi har gjort tidligere i prosessen fordi man har vært rask og løs med cowboy kodingen :cowboy_hat_face: